src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -6751,7 +6752,7 @@ namespace VariableSpeedCoils {
 
 	//     NOTICE
 
-	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
+	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
```

Comment:
By me taking out the symbol entirely...please hold and I'll probably submit a PR to fix the copyright things...